WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -27,13 +35,24 @@ final class ProductCategoryListViewController: UIViewController, GhostableViewCo
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        configureSearchBar()
+        configureClearSelectionButton()
         registerTableViewCells()
         configureTableView()
         configureViewModel()
         handleSwipeBackGesture()
     }
 }
 
+// MARK: - Configuration to customize the list
+//
+extension ProductCategoryListViewController {
+    struct Configuration {
+        var searchEnabled: Bool = false
+        var clearSelectionEnabled: Bool = false
```

Comment:
I think that we can skip explicitly mentioning the type here. i.e. Remove `Bool`.